### ISSUE TITLE
Unable to install version 1.9.10

### ISSUE THREAD

**Issue by pjotrsavitski** [hechoendrupal/drupal-console#4343]
### Unable to install version 1.9.10

[ *ISSUE-GROUP* ] console

### Problem/Motivation

It is impossible to install the latest version 1.9.10 and version 1.9.9 is installed, although the later one is listed as possible update.

### How to reproduce
Include steps related how to reproduce.

**Details to include:**
- Drupal: 9.4.7
-  Drupal console: 1.9.9 (current one)
- Console Launcher version: no idea about this one
- How to reproduce: composer update drupal/console and composer outdated "drupal/*"

### Solution
It seems that it is due to the dependency that does not yet have a tagged version released: `console-core ^1.9.8`

Most probably make sure that dependency version  is released.

**Comment by LOBsTerr** [comment]
It would be nice to have some additional info ? 
What errors do you have from composer

**Comment by pjotrsavitski** [comment]
@LOBsTerr It does not really give any errors, just does not install the current latest version. I'll try to add some detailed information here.

**Long shory short. Version 1.9.8 for drupal/console-core has not been tagged/released. This one prevents the latest version of the package from being considered for installation, unless the concrete versions is specified, which makes composer report an issue**.

This is what the `composer outdated "drupal/*"` command shows (it does list the latest version):
```
- patch or minor release available - update recommended
- major release available - update possible
drupal/console 1.9.9 1.9.10 The Drupal CLI. A tool to generate boilerplate c...
```

Running `composer update drupal/console` did update the older version to **1.9.9**, but it does not go beyond it.
I did try running `composer why drupal/console` and other commands, but it did not really give much information to work with.
The real reason could be seen when setting a concrete version in composer.json file like `"drupal/console": "1.9.10"`. The update command output was:
```
Gathering patches for root package.
No patches supplied.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires drupal/console 1.9.10 -> satisfiable by drupal/console[1.9.10].
    - drupal/console 1.9.10 requires drupal/console-core ^1.9.8 -> found drupal/console-core[dev-feature/D9-support, dev-feature/update-php-version, dev-master, dev-tag-1-9-7, 1.0.0-rc1, ..., 1.9.7] but it does not match the constraint.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

This lead me to believe that the issue is with `drupal/console-core` dependency. Running `composer why drupal/console-core` explains that it is a direct dependency of just this package. I did pay a visit to the  [dependency packagist page](https://packagist.org/packages/drupal/console-core) and found out that the latest listed version, apart from **dev-master**, is **1.9.7**. This more or less means that there is no such version as **1.9.8**.
Visiting GitHub repository repository also shows that the latest tagged release is **1.9.7**. My guess that tagging a new release has been forgotten or something like that.

I hope that help.
